.gitignore

```diff
@@ -3,3 +3,9 @@ node_modules
 fqdn.env
 /docker/certs
 qrcode-svg/
+rtc_config.json
+turnserver.conf
+logs/*
+*.orig
+*.log
+letsencrypt/*
```

Comment:
We don't need lines 8-10. Lines 3-5 could also be deleted but maybe in a separate commit

Reply:
I have them removed now.